embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/arch/schema/SendMode.kt

```diff
@@ -0,0 +1,17 @@
+package io.embrace.android.embracesdk.internal.arch.schema
+
+import io.embrace.android.embracesdk.internal.arch.schema.SendMode.DEFAULT
+import io.embrace.android.embracesdk.internal.arch.schema.SendMode.DEFER
+import io.embrace.android.embracesdk.internal.arch.schema.SendMode.IMMEDIATE
+
+public enum class SendMode {
+    DEFAULT, IMMEDIATE, DEFER
```

Comment:
Some Kdoc here would be helpful in explaining what each mode does

Reply:
Yeah good call. I was going to wait until we do the refactor but I shouldn't wait for that.